includes/Media.php

```diff
@@ -346,7 +367,12 @@ public function rest_api_init() {
 				'schema'          => [
 					'description' => __( 'Media source. ', 'web-stories' ),
 					'type'        => 'string',
-					'enum'        => [ 'editor', 'poster-generation', 'video-optimization' ],
+					'enum'        => [
+						'editor',
+						'poster-generation',
+						'video-optimization',
+						'source-video',
```

Comment:
@spacedmonkey What does `source-video` mean here? Why was a new term needed? The name is not ideal here either :/

Reply:
Source video, is a term added to the original video. It is used to hide the original video.

Reply:
Ah, got it!